### PR TITLE
Make fact that click/contextmenu are not compat mouse events normative

### PR DESCRIPTION
**Pull request by patrickhlauke**
x-ref https://github.com/w3c/pointerevents/issues/405

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/pointerevents/pull/418.html" title="Last updated on Oct 20, 2021, 9:06 AM UTC (d7a2770)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/pointerevents/418/76e270a...d7a2770.html" title="Last updated on Oct 20, 2021, 9:06 AM UTC (d7a2770)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -434,19 +434,19 @@ <h3>Attributes and default actions</h3>
                                     <td>Yes</td>
                                     <td>Yes</td>
                                     <td>Varies: when the pointer is primary, all default actions of the <code>mousedown</code> event
-                                        <br>Canceling this event also sets the <code>PREVENT MOUSE EVENT</code> flag for this <code>pointerType</code>, which prevents subsequent firing of certain <a>compatibility mouse events</a>.</td>
+                                        <br>Canceling this event also sets the <code>PREVENT MOUSE EVENT</code> flag for this <code>pointerType</code>, which prevents subsequent firing of <a>compatibility mouse events</a>.</td>
```
Comment (mustaqahmed) [inline index.html]: When I just took a fresh look at this, it seems to me that `PREVENT MOUSE EVENT` flag is just an algorithmic details that is fully defined in Sec11.  I am proposing to remove it from here and also from the note in 4.2.3.

So this para would look like this:
> Canceling this event also prevents subsequent firing of compatibility mouse events.

Wdyt?

File: index.html [hunk 2]
```diff
@@ -1142,8 +1142,8 @@ <h2><dfn>Populating and maintaining the coalesced and predicted events lists</df
         <h1><dfn data-lt="compatibility mouse events">Compatibility mapping with mouse events</dfn></h1>
         <p>The vast majority of web content existing today codes only to Mouse Events. The following describes an algorithm for how the user agent MAY map generic pointer input to mouse events for compatibility with this content.</p>
         <p>The compatibility mapping with mouse events are an OPTIONAL feature of this specification. User agents are encouraged to support the feature for best compatibility with existing legacy content. User agents that do not support compatibility mouse events are still encouraged to support the <code>click</code> and <code>contextmenu</code> events (see the note below).</p>
+        <p>The <code>click</code> event, defined in [[UIEVENTS]], and the <code>contextmenu</code> event are not considered <a>compatibility mouse events</a> as they are typically tied to user interface activation and are fired from other input devices, like keyboards.</p>
```
Comment (mustaqahmed) [inline index.html]: We have just added `contextmenu` event to UIEvents, so we can now clean this up a bit:

> The `click` and <code>contextmenu</code> events, defined in [[UIEVENTS]], are not...

File: index.html (hunk 1 above)
Comment (patrickhlauke) [reply index.html]: yup, makes sense. done.

File: index.html (hunk 2 above)
Comment (patrickhlauke) [reply index.html]: done. also tweaked the end of that sentence slightly for better readability/understandability

**Comment by patrickhlauke** [comment]
As the changes here were discussed fully in our last meeting and are quite non-controversial, I'm just going to blaze ahead and merge it. Shout (retrospectively) if there was something that needed looking at further here.

**Comment by smaug----** [comment]
click is tested in many tests and contextmenu https://searchfox.org/mozilla-central/source/testing/web-platform/tests/pointerevents/pointerevent_contextmenu_is_a_pointerevent.html
And this pr didn't really change any of that.

**Pull request merged**